onnxruntime/core/providers/nuphar/kernel.h

```diff
@@ -107,8 +108,9 @@ class NupharKernelState {
   NUPHAR_VERSIONED_OP(MatMul, 1, 8, DataTypeImpl::AllIEEEFloatTensorExceptHalfTypes())       \
   NUPHAR_OP(MatMul, 9, DataTypeImpl::AllIEEEFloatTensorExceptHalfTypes())                    \
   NUPHAR_OP(Max, 8, DataTypeImpl::AllFixedSizeTensorTypes())                                 \
-  DISABLE_MACRO(NUPHAR_VERSIONED_OP(MaxPool, 1, 7, DataTypeImpl::AllFixedSizeTensorTypes())) \
-  DISABLE_MACRO(NUPHAR_OP(MaxPool, 8, DataTypeImpl::AllFixedSizeTensorTypes()))              \
+  NUPHAR_VERSIONED_OP(MaxPool, 1, 7, DataTypeImpl::AllFixedSizeTensorTypes())                \
+  NUPHAR_VERSIONED_OP(MaxPool, 8, 9, DataTypeImpl::AllFixedSizeTensorTypes())                \
+  NUPHAR_OP(MaxPool, 10, DataTypeImpl::AllFixedSizeTensorTypes())                            \
```

Comment:
>MaxPool [](start = 12, length = 7)

Seems there's opset11 for this op now.

Reply:
Hmm, I didn't see opset11 for MaxPool in CPU's implementation:

https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/core/providers/cpu/cpu_execution_provider.cc#L271